### DISCUSSION TITLE
Can we disable the checkbox rendered by the TableSelectionCell component?

### DISCUSSION THREAD

**Discussion by nathaniellee** [microsoft/fluentui#31719] [Q&A]
I didn't see a disable prop [here](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-table/library/src/components/TableSelectionCell/TableSelectionCell.types.ts). Is there a way to pass a disable prop in? Or do we need to just manually construct checkboxes in the table if we have a use case where some rows need to be disabled?

**Comment by layershifter** [comment]
You can use slots to do this:

```ts
<TableSelectionCell checkboxIndicator={{ disabled: true }} />
<TableSelectionCell radioIndicator={{ disabled: true }} />
```

^ should do the thing 🐱
